src/corporacreator/preprocessors/cy.py

```diff
@@ -1,12 +1,38 @@
-def cy(client_id, sentence):
-    """Cleans up the passed sentence, removing or reformatting invalid data.
+# compile the set of the alphabet just once
+alphabet = set([' ', 'a','á', 'à', 'â', 'ä', 'b', 'c',
+                'd', 'e', 'é', 'è', 'ê', 'ë', 'f', 'g',
+                'h', 'i', 'í', 'ì', 'î', 'ï', 'j', 'l',
+                'm', 'n', 'o', 'ó', 'ò', 'ô', 'ö', 'p',
+                'r', 's', 't', 'u', 'ú', 'ù', 'û', 'ü',
+                'w', 'ẃ', 'ẁ', 'ŵ', 'ẅ', 'y', 'ý', 'ỳ',
+                'ŷ', 'ÿ'])
```

Comment:
Should include upper and lower case letters.

Reply:
I would vote that we keep the alphabet lowercase, and then lowercase the sentence when 
we check whether or not the sentence is a subset of the alphabet, as in here:

```
     if set(sentence.lower()).issubset(alphabet):
```

To me, this seems easier for a contributer to be able to quickly read the alphabet.

Reply:
Casing is local specific, e.g. [Turkish](https://en.wikipedia.org/wiki/Dotted_and_dotless_I).

So doing what you suggest would mean that we have to specify which local the code should be run in. We should just have to run the code and have it work.

Reply:
@kdavis-mozilla --- lowercasing removed in latest commit